recipes/qt.py/meta.yaml

```diff
@@ -32,7 +32,7 @@ about:
   home: http://github.com/mottosso/Qt.py
   license: MIT
   license_family: MIT
-  license_file: LICENSE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Thanks for adding this. Can you also raise the issue upstream about including the file in the source distribution? Adding a MANIFEST.IN should solve this.

Reply:
Sure! Thanks @isuruf 😄

Reply:
@isuruf would it be fine to merge this without waiting for AppVeyor? I would really like to start using this as soon as possible.